app/src/main/res/values-v21/styles.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="PumpkinTheme" parent="Theme.AppCompat.Light.DarkActionBar">
```

Comment:
Does it mean that if the device if lollipop ready, it will use PumpkinTheme.Material whereas if it's lower than lolipop, it will automatically use PumpkinTheme?